src/test/regress/expected/multi_subquery.out

```diff
@@ -911,8 +911,16 @@ RESET citus.coordinator_aggregation_strategy;
 SELECT t1.event_type FROM events_table t1
 GROUP BY t1.event_type HAVING t1.event_type > corr(t1.value_3, t1.value_2 + (SELECT t2.value_2 FROM users_table t2 ORDER BY 1 DESC LIMIT 1))
 ORDER BY 1;
-ERROR:  result "68_1" does not exist
-CONTEXT:  while executing command on localhost:xxxxx
+ event_type
```

Comment:
might be nice to add this query as well: https://github.com/citusdata/citus/pull/3709/files#diff-14b324e6d5be7acd78f5be06eabddd3fR113